politeiawww/api/www/v1/v1.go

```diff
@@ -724,6 +726,16 @@ type ProposalDetailsReply struct {
 	Proposal ProposalRecord `json:"proposal"`
 }
 
+// BatchProposals is used to request the details of multiple proposals.
```

Comment:
```suggestion
// BatchProposals is used to request the details of multiple proposals. The
// returned proposals do not include the proposal files.
```

This is non-obvious behavior so its better to explicitly state it.